data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -434,6 +434,58 @@ spec:
                           - high_performance
                           type: string
                       type: object
+                    powervs:
+                      description: PowerVS is the configuration used when installing
+                        on IBM Power VS.
+                      properties:
+                        imageName:
+                          description: ImageName defines the boot image name for the
+                            instance.
+                          type: string
+                        keypairname:
+                          description: KeyPairName is the name of an SSH key pair
+                            stored in the Power VS Service Instance
+                          type: string
+                        memory:
+                          description: Memory defines the memory in GB for the instance.
+                          type: integer
+                        name:
+                          description: Name is the name of the instance
+                          type: string
+                        networkIDs:
+                          description: NetworkIDs defines the network IDs of the instance.
+                          items:
+                            type: string
+                          type: array
+                        procType:
+                          description: ProcType defines the processor sharing model
+                            for the instance.
+                          type: string
+                        processors:
+                          description: 'Processors defines the processing units for
+                            the instance. @TODO:'
+                        serviceinstance:
+                          description: ServiceInstance is Service Instance to install
+                            into.
+                          type: string
+                        sysType:
+                          description: SysType defines the system type for instance.
+                          type: string
+                        volumeIDs:
+                          description: VolumeIDs is the list of volumes attached to
+                            the instance.
+                          items:
+                            type: string
+                          type: array
+                      required:
+                      - imageName
+                      - keypairname
+                      - memory
+                      - name
+                      - networkIDs
+                      - processors
+                      - serviceinstance
```

Comment:
can't we have well defined defaults for most of these fields so that most of them are optional ?

Reply:
same comment as above. we do have defaults though